bool inflateMap
float64 inflateRadius
---
bool success